packages/config/package.json

```diff
@@ -11,7 +11,8 @@
   "type": "module",
   "exports": {
     ".": "./dist/library/index.js",
-    "./register": "./dist/register/index.js"
+    "./register": "./dist/register/index.js",
+    "./cli": "./dist/cli/index.js"
```

Comment:
should this be `/node`?

Reply:
yeah I was torn between the two, but now agree `node` fits better - changed it for both config and world exports